app/views/pulitzer/post_type_content_element_types/_form_fields.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= f.label :label %>
+<%= f.text_field :label %>
+<%= f.label :content_element_type_id, "Type" %>
+<%= f.collection_select :content_element_type_id, Pulitzer::ContentElementType.all, :id, :name, {},
+{ data: { "revealer" => true, "sub-type" => "FormFieldRevealer", "revealer-children-id" => dom_id(ptcet, :revealer) }} %>
```

Comment:
I remembered that we have a helper here: https://github.com/edraut/hooch/blob/master/lib/hooch/hooch_helper.rb#L71-L77

don't worry about using it now, I think the helper needs some improvement. We can add it later. I think it needs that idea you had where it will take the other options and merge them for you. Just a reference for future.